src/services/room.service.js

```diff
@@ -0,0 +1,31 @@
+const { ApiError } = require('../exceptions/api.error');
+const { Room } = require('../models/Room.model');
+
+const create = async (name) => {
+  const newRoom = await Room.create({ name });
+
+  return newRoom;
+};
+
+const remove = async (id) => {
+  await Room.destroy({ where: { id } });
+};
+
+const rename = async (id, newName) => {
+  if (!newName) {
+    throw ApiError.badRequest('new name required');
+  }
+
+  const renamedRoom = await Room.findOne({ where: { id } });
```

Comment:
It's a good practice to check if the room was found before attempting to rename it. If `Room.findOne` returns `null`, trying to access `name` on it will throw an error. Consider adding a check to handle the case where the room is not found.